ckanext/nhm/theme/templates/about/citation.html

```diff
@@ -45,19 +45,30 @@ <h3>2. Resources</h3>
     </div>
 
     <h3>3. Records</h3>
-    <p>Records are versioned, so you can either cite the most up-to-date version of the record or the version as it was when you initially retrieved it. Both URLs can be found in the 'Cite this as' section at the bottom of a record page.</p>
+    <p>Records from the <i>Specimens</i> resource are versioned, so you can either cite the most up-to-date version of the record or the version as it was when you initially retrieved it. Both URLs can be found in the 'Cite this as' section at the bottom of a record page.</p>
```

Comment:
Lol, sorry, this is my bad. All records are versioned, it's just that the specimen records have the special `/object/` URLs. You can get a specific version of any record by going to `https://data.nhm.ac.uk/dataset/<u>dataset-id</u>/resource/<u>resource-id</u>/record/<u>record-id</u>/<u>version</u>` e.g. https://data.nhm.ac.uk/dataset/join-the-dots-collection-level-descriptions/resource/660de433-447b-40ad-9aaf-77c98f8c2ba9/record/4633/1652709718000.